components/list/list-item-generic-layout.js

```diff
@@ -186,7 +198,7 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 			<slot name="control-action" class="d2l-cell" data-cell-num="3"></slot>
 			<slot name="control" class="d2l-cell" data-cell-num="4"></slot>
 			<slot name="actions" class="d2l-cell" data-cell-num="6"></slot>
-			<slot name="content" @focus="${this._preventFocus}"></slot>
+			<slot name="content" class="d2l-cell" data-cell-num="7" @focus="${!this.noPrimaryAction ? this._preventFocus : null}"></slot>
```

Comment:
Adding `d2l-cell` class and `data-cell-num` includes this element in the search for focusables when using arrow keys to move among the cells/focusables within cells.